democracy/views/section_comment.py

```diff
@@ -6,20 +6,28 @@
 from rest_framework.serializers import get_validation_error_detail
 from rest_framework.settings import api_settings
 
-from democracy.models import SectionComment
+from democracy.models import SectionComment, Label
 from democracy.views.comment import COMMENT_FIELDS, BaseCommentViewSet, BaseCommentSerializer
+from democracy.views.label import LabelSerializer
 from democracy.pagination import DefaultLimitPagination
-from democracy.views.utils import filter_by_hearing_visible
+from democracy.views.utils import filter_by_hearing_visible, JSONPKRelatedField
 
 
 class SectionCommentCreateSerializer(serializers.ModelSerializer):
     """
     Serializer for comments creation.
     """
+    label = JSONPKRelatedField(
```

Comment:
Weird indentation. (The kwargs should be indented 8 spaces; the trailing `)` should be flush with `label = `.)